### DISCUSSION TITLE
Hypothetical Scenario

### DISCUSSION THREAD

**Discussion by macacob** [frntc/SIDKick-pico#78] [General]
If one were to solder the TL072 backwords, and then install it in a C64 and turn it on, what other components of the SKPico with onboard DAC might get fried?  Asking for a friend. (Thankfully no damage to the C64)

**Comment by frntc** [comment]
Hm... I have no empirical values ;-) I think there's a good chance that only the TL072 is damaged. Have you tried replacing it (it's probably fried, at least you, well your friend, swapped +5V and GND)

**Reply by macacob** [reply]
That was my first guess as well after looking at the schematics of the TL072.  I desoldered it and replaced it with a new one.  I was able to get to the menu but no sound through PWM (I did solder the jumper to allow the DAC model to play through PWM, I made that mistake on my first build). I have gone back over all of my soldering work and reflowed everything and it looks clean. I have more to build so I'll probably set this one aside for now but and build those more carefully, but it drives me nuts not knowing why something isn't working.

**Reply by frntc** [reply]
You need to take the DAC-firmware, there's no PWM sound on this one! The opamp is there to drive the audio through the mainboard :-)

**Reply by macacob** [reply]
Hmm, good suggestion. I will go back and triple check what I installed on the pico, but I am 99% sure it is the same firmware I have on prior builds which are working.

**Reply by macacob** [reply]
Soooo, both the SKPico board and the Pico clone board are faulty. the RGB Pico clone does not work with my known good SKpico build, and my known good RGB Pico clone will not work with my faulty SKpico build.  not a firmware issue I guess, something really did get damaged.

**Reply by frntc** [reply]
Is there just no sound, or does the SKpico not even launch the config too with the known-good Pico clone?

**Reply by macacob** [reply]
With a known good Pico there is no sound but the config launches.

**Reply by frntc** [reply]
Then the CBTD3861 should be good, but the DAC is not...
